batch/batch/server/database.py

```diff
@@ -135,6 +136,49 @@ def __init__(self, db):
     async def get_records_where(self, condition):
         return await super().get_records(condition)
 
+    async def find_records(self, user, complete=None, success=None, deleted=None, attributes=None):
+        sql = "select batch.* from {self.name} as batch"
+        values = []
+        joins = []
+        wheres = []
+        havings = []
+        groups = []
+
+        values += user
+        wheres += "job.user = %s"
+        if deleted is not None:
+            if deleted:
+                wheres += "batch.deleted"
+            else:
+                wheres += "not batch.deleted"
+        if complete is not None:
+            condition = "not batch.is_open and batch.n_completed = batch.n_jobs"
```

Comment:
The  `is_open` will conflict with my atomic batch creation PR. I bet yours will go in first though so I'll end up changing it later.